docs/documentation/updating-the-kit.md

```diff
@@ -1,6 +1,6 @@
-# Updating your Prototype Kit
+# Update your Prototype Kit
 
-How to to update your prototype and get help from the GOV.UK Prototype Kit team.
+How to update your prototype and get help from the GOV.UK Prototype Kit team.
```

Comment:
Completely optional: we could add something here about how it's good to update to the latest version if at all possible to get access to security patches and new features. But that might be one for a future iteration of this doc.

Reply:
That sounds like a good idea, I'll turn it into a ticket.

Reply:
https://github.com/alphagov/govuk-prototype-kit/issues/1193